observability/otel/otel_rn/otel-rn-past-releases.adoc

```diff
@@ -8,6 +8,74 @@ toc::[]
 
 include::modules/otel-product-overview.adoc[leveloffset=+1]
 
+[id="otel-rn-3-2"]
+== Release notes for {OTELName} 3.2
+
+The {OTELName} is provided through the {OTELOperator}.
+
+////
+[id="otel-rn_3-2_cves"]
+=== CVEs
+
+This release fixes link:https://access.redhat.com/security/cve/cve-202?-?????[CVE-202?-?????].
+////
+
+[id="otel-rn_3-2_technology-preview-features"]
+=== Technology Preview features
```

Comment:
Jw if you want to include a boilerplate about what TP is. In NetObserv Tech Preview, I copied the OCP Release notes: https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-technology-preview-tables

Reply:
Thank you for sharing. I include the required snippet there and would like to keep it minimalist.

```
:FeatureName: The Tempo monolithic deployment
include::snippets/technology-preview.adoc[leveloffset=+1]
```